scribejava-apis/src/test/java/com/github/scribejava/apis/examples/AsanaExample.java

```diff
@@ -0,0 +1,85 @@
+package com.github.scribejava.apis.examples;
+
+import com.github.scribejava.apis.Asana20;
+import com.github.scribejava.core.builder.ServiceBuilder;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.OAuthRequest;
+import com.github.scribejava.core.model.Response;
+import com.github.scribejava.core.model.Verb;
+import com.github.scribejava.core.oauth.OAuth20Service;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.concurrent.ExecutionException;
+
+public class AsanaExample {
+
+    private static final String PROTECTED_RESOURCE_URL = "https://app.asana.com/api/1.0/users/me";
+
+    private AsanaExample() {
+
+    }
+
+    public static void main(String... args) throws IOException, InterruptedException, ExecutionException {
+        final String apiKey = "your client id";
+        final String apiSecret = "your client secret";
+        final String secretState = "secret" + new Random().nextInt(999_999);
```

Comment:
became useless.
It should be set via service.state(secretState) method.
I will add myself this line of code